02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,29 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>My friggen page</title>
 </head>
 <body>
+  <h1>Dan Cumming</h1>
+  <section><em>1300 E Riverside Dr.</em></section>
```

Comment:
I would recommend putting tags of block element in their own line: 
```html
<section>
   <em>1300 ... </em>
</section>
```
That makes it easier to find errors and avoids mistakes with missing closing tags etc.